Fix window being focused by default (#7899)
alacritty/alacritty: CHANGELOG.md

The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).

> Notable changes to `alacritty_terminal` crate should be documented in its
> [CHANGELOG](./alacritty_terminal/CHANGELOG.md).

Reviewer: This is a user-facing document, however this is worded in a developer-centric fashion.

This would probably be better:

```suggestion
Notable changes to the `alacritty_terminal` crate are documented in its
[CHANGELOG](./alacritty_terminal/CHANGELOG.md).
```

Arguably the same could be said about the documentation about the sections, but that is a bit older.

## 0.14.0-dev

### Fixed

> - New window sometimes being treated as focused when it's not on Wayland

Reviewer: ```suggestion
- New window being treated as focused when it's not on Wayland
```

Author: This bug is really hard to trigger though, I'm not sure you can even face it, but ok.

Reviewer: Yeah it's just kinda unnecessary. Because people who aren't running into the bug just don't care. How frequently a bug occurred isn't really relevant. The people that ran into it know it's fixed, the people that never ran into it don't care either way.
